static/src/javascripts/projects/common/modules/identity/api.js

```diff
@@ -0,0 +1,284 @@
+// @flow
+
+/* global escape:true */
+import { ajax } from 'lib/ajax';
+import config from 'lib/config';
+import { getCookie as getCookieByName } from 'lib/cookies';
+import mediator from 'lib/mediator';
+import { local } from 'lib/storage';
+import { mergeCalls } from 'common/modules/asyncCallMerger';
+
+/**
+ * Left this as an object as there are only static methods
+ * We'll need to change this once there is some state change
+ * TODO(jamesgorrie): Allow this to show policies too (not needed yet)
+ */
+
+let userFromCookieCache = null;
+
+const Id = {
+    cookieName: 'GU_U',
+    signOutCookieName: 'GU_SO',
+    fbCheckKey: 'gu.id.nextFbCheck',
+    lastRefreshKey: 'identity.lastRefresh',
+    idApiRoot: null,
+};
+
+export const init = () => {
```

Comment:
It would be great if you could add return types for all exported functions 🙏  This would give us greater confidence in this migration

Reply:
I will delete the types but no the descriptions since I think are useful.

Reply:
I rarely find the descriptions useful, they tend to repeat the name of the function or parameter. I'd be happy for them to be deleted.